salt-master/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM centos:7.6.1810
```

Comment:
Let's move this into an `images/salt-master` directory or similar.

Reply:
Done